packages/fxa-settings/src/pages/Signup/container.tsx

```diff
@@ -210,7 +210,7 @@ const SignupContainer = ({
         // If enabled, add in V2 key stretching support
         let credentialsV2 = undefined;
         let passwordV2 = undefined;
-        if (keyStretchExp.queryParamModel.isV2()) {
+        if (keyStretchExp.queryParamModel.isV2(config)) {
```

Comment:
Good catch on the missing dep! 

For the nit, I sorta like passing the config, because the resulting code feels more obvious to (ie I know exactly where the feature flag comes form...) So I might ignore, unless you would really rather have the other way.